File: Codewars/8kyu/8kyu_find_the_smallest_integer_in_array/src/main.py
def find_smallest_int(arr):
    return min(arr)

print find_smallest_int([78, 56, 232, 12, 11, 43]) == 11
print find_smallest_int([78, 56, -2, 12, 8, -33]) == -33
print find_smallest_int([0, 1 - 2**64, 2**64]) == 1 - 2**64

File: Codewars/8kyu/8kyu_find_the_smallest_integer_in_array/src/main.js
function mysolution(args) {
    return Math.min(...args)
}

function othersolution(args) {
    return Math.min.apply(null, args)
}

const findSmallestInt = othersolution

console.log(findSmallestInt([78,56,232,12,8]) === 8);
console.log(findSmallestInt([78,56,232,12,18]) === 12);
console.log(findSmallestInt([78,56,232,412,228]) === 56);
console.log(findSmallestInt([78,56,232,12,0]) === 0);
console.log(findSmallestInt([1,56,232,12,8]) === 1);